### PR TITLE
Reorganized examples to build a progressive tutorial

### PR DESCRIPTION
**Pull request by fgm**
Brought the simple example from PR #27 to add a query and field under examples/ as step 1 of a progressive tutorial, and move the existing example there too as step 3.

### CODE REVIEW

File: examples/step1_query/graphql_example_query.services.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+services:
+  graphql_example_query.schema_provider:
+    class: 'Drupal\graphql_example_query\SchemaProvider\LatestUserSchemaProvider'
+    arguments:
+      - '@typed_data_manager'
+      - '@graphql.type_resolver'
+    tags:
+        - name: 'graphql_schema_provider'
```
Comment (RoySegall) [inline examples/step1_query/graphql_example_query.services.yml]: Wrong indentation, no?

File: examples/step1_query/src/GraphQL/Field/Root/LatestUserField.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    $entityStorage = $entityTypeManager->getStorage('user');
+    $query = $this->container->get('entity.query');
+    $uids = $query
+      ->getAggregate('user')
+      ->aggregate('uid', 'MAX')
+      ->execute();
+    $uid = reset($uids);
+    return $entityStorage->load($uid['uid_max']);
+  }
+}
```
Comment (RoySegall) [inline examples/step1_query/src/GraphQL/Field/Root/LatestUserField.php]: An empty line should be at the end of the class.

File: examples/step1_query/graphql_example_query.services.yml (hunk 1 above)
Comment (fgm) [reply examples/step1_query/graphql_example_query.services.yml]: Right: funny that it even works.

**Comment by fgm** [comment]
More or less a duplicate of #27

**Pull request closed without merging**